source/_docs/installation.markdown

```diff
@@ -131,3 +131,124 @@ For more details about `hass`, please refer to the [tools section](/docs/tools/h
 If you're running a Linux-based platform, we suggest you follow the [VirtualEnv instructions](/docs/installation/virtualenv/) to avoid using `root`.
 
 You may need to install additional libraries depending on the platforms/components you want to use.
+
+## {% linkable_title Characteristics of different install methods for Home Assistant %}
+
+You always get the full Home Assistant expirience despite the installation method you choose to use.
+How you would like to run Home Assistant is a choice of taste, the setup you already have, knowlede and what you want to accomplish with home assistant.
+
+<p class='note'>
+Add-Ons provide an easy way to add more functionality to your setup through the gui.
+Add-Ons in Hass.io are just docker containers containing some software.
```

Comment:
Basically, correct, nevertheless, they do have more integrations possibilities in comparison to plain Docker.

```suggestion
Add-ons in Hass.io are essentially just docker containers containing some software.
```